.github/workflows/npm-typescript.yml

```diff
@@ -0,0 +1,53 @@
+name: NPM
```

Comment:
Can we name it in a way indicating this is a workflow related to `typescript`? And I think we should do the same for `npm-contracts.yml`

Reply:
Yah, I missed that. Addressed in [663fa76](https://github.com/keep-network/tbtc-v2/pull/341/commits/663fa762352154f8d331786de9ebc969e76c332d).